plugin/src/main/java/io/snyk/eclipse/plugin/properties/PreferencesPage.java

```diff
@@ -72,13 +57,17 @@ protected void createFieldEditors() {
     addField(new StringFieldEditor(Preferences.ADDITIONAL_PARAMETERS, "Additional Parameters:", getFieldEditorParent()));
     addField(new StringFieldEditor(Preferences.ADDITIONAL_ENVIRONMENT, "Additional Environment:", getFieldEditorParent()));
 
+    addField(space());
+    addField(new BooleanFieldEditor(Preferences.MANAGE_BINARIES_AUTOMATICALLY, "Update and install Snyk binaries automatically", getFieldEditorParent()));
+    addField(new FileFieldEditor(Preferences.LS_BINARY_KEY, "Snyk LS Path:", getFieldEditorParent()));
```

Comment:
Please make this `Snyk Language Server Path`. Feedback from customers was that they don't really know what `LS` stands for.